packages/twenty-front/src/modules/activities/emails/right-drawer/components/RightDrawerThread.tsx

```diff
@@ -1,29 +1,47 @@
 import React from 'react';
 import styled from '@emotion/styled';
+import { useRecoilValue } from 'recoil';
 
+import { MessageThread } from '@/activities/emails/components/MessageThread';
 import { ThreadHeader } from '@/activities/emails/components/ThreadHeader';
+import { mockedMessagesByThread } from '@/activities/emails/mocks/mockedThreads';
+import { viewableThreadState } from '@/activities/emails/state/viewableThreadState';
 
 const StyledContainer = styled.div`
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
   height: 100%;
-  justify-content: space-between;
+  justify-content: flex-start;
   overflow-y: auto;
   position: relative;
 `;
 
 export const RightDrawerThread = () => {
```

Comment:
Make sure to add EmailThread everywhere in the namings, this will ease coding experience